packages/perseus/src/widgets/label-image/score-label-image.ts

```diff
@@ -53,21 +54,14 @@ function scoreLabelImage(
             scoringData.markers[i].answers,
         );
 
-        if (score.hasAnswers) {
-            numAnswered++;
-        }
-
         if (score.isCorrect) {
             numCorrect++;
         }
     }
 
-    // We expect all question markers to be answered before grading.
-    if (numAnswered !== userInput.markers.length) {
-        return {
-            type: "invalid",
-            message: null,
-        };
+    const validationError = validateLabelImage(userInput);
+    if (validationError) {
+        return validationError;
```

Comment:
I think this is going to need to be moved to the beginning of this function. It shouldn't have any impact on the code because we're iterating over the markers twice anyway.

Reply:
Moved! Thanks for pointing this out :)